### ISSUE TITLE
Update V1.9

### ISSUE THREAD

**Issue by LesnterFinester** [nipkownix/re4_tweaks#505]
Game was working perfectly till now, A prompt showed up to auto update to the latest of Re4Tweaks. Accepted it. Restart, now this happened 
![image](https://user-images.githubusercontent.com/81026023/226774642-77571407-3349-454a-bcbf-b4e054bc2406.png)

cant play the game anymore pass the Oversized "LOADING"

**Comment by Edu-Nov** [comment]
After the update, after loading a save the screen goes black, audio and inputs still works, but the screen is just black,

Edit: Game works normally after switching to Vulkan render

**Comment by nipkownix** [comment]
Do you have a fully updated version of the RE4 HD Project installed? Can you check if the files ``doorse012.xsb`` and ``doorse012.xwb`` exist in your ``\Resident Evil 4\BIO4\snd`` directory?

**Comment by LesnterFinester** [comment]
Yes, recently bought the game on monday/ or last saturday. Downloaded the
most recent version available in their official website and the re4 tweaks
(v1.8 at that time) game worked flawlessly until today, a prompt pop up for
the v1.9 accepted it. Game starts normaly, starting screen menu is normal
size but sub menu are huge Same as loading screen which will just hang and
cause the game to not respond.

**Comment by nipkownix** [comment]
Have you checked the two files I mentioned?

**Comment by LesnterFinester** [comment]
The new version has fixed the problem. (v.1.9.1)
thank you. I can resume my playthrough at the best possible experience It can have.

**Comment by nipkownix** [comment]
I'm glad it is working now. Thanks for letting me know!
